modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/model/DDMFormField.java

```diff
@@ -53,6 +50,14 @@ public DDMFormField(DDMFormField ddmFormField) {
 			addDDMFormFieldRule(new DDMFormFieldRule(ddmFormFieldRule));
 		}
 
+		DDMFormFieldOptions ddmFormFieldOptions =
+			ddmFormField.getDDMFormFieldOptions();
+
+		if (ddmFormFieldOptions != null) {
+			setDDMFormFieldOptions(
+				new DDMFormFieldOptions(ddmFormFieldOptions));
+		}
```

Comment:
why did you move this down? can we maintain this code in the original line?

Reply:
Yes, we can! I tried to put closer to the others null verification and sets we have on this method.

Reply:
lets maintain.

Reply:
Yes ma'am